website/src/pages/api/update_task.ts

```diff
@@ -33,7 +33,11 @@ const handler = withoutRole("banned", async (req, res, token) => {
 
   const taskId = (registeredTask.task as Prisma.JsonObject).id as string;
 
-  await oasstApiClient.ackTask(taskId, registeredTask.id);
+  try {
+    await oasstApiClient.ackTask(taskId, registeredTask.id);
+  } catch (e) {
+    console.error(e);
+  }
```

Comment:
This try catch is required. When retrying the task, the backend will deny it because the task is already acked.

Reply:
I have made ACK idempotent if it is called with the same frontend_message_id ... https://github.com/LAION-AI/Open-Assistant/commit/3ba5bfe40c6f37b3e2e5e4a28082fb6661383bff